pontoon/settings/base.py

```diff
@@ -515,11 +516,12 @@ def _default_from_email():
             "js/lib/jquery.color-2.1.2.js",
             "js/lib/nprogress.js",
             "js/main.js",
+            "js/themeSwitcher.js",
         ),
         "output_filename": "js/base.min.js",
     },
     "translate": {
-        "source_filenames": ("translate.js",),
+        "source_filenames": ("translate.js", "js/themeSwitcher.js"),
```

Comment:
We don't need that code in the translate view. We should re-implement the logic in React.